op-chain-ops/foundry/allocs.go

```diff
@@ -83,6 +83,14 @@ func (d *ForgeAllocs) Copy() *ForgeAllocs {
 }
 
 func (d *ForgeAllocs) UnmarshalJSON(b []byte) error {
+	type container struct {
+		Accounts json.RawMessage
+	}
```

Comment:
Could we use a variable of type `json.RawMessage` here instead of using the `container` struct with a single field? Or are you planning to add more fields to this struct later?

Reply:
The incoming JSON looks like this:

```json
{ "Accounts": ...allocs }
```

So I need the container struct to pull out the `Accounts` field so it can be further unmarshaled later on.

Reply:
Turns out this needs to marshal without the `Accounts` container - updated to remove that on marshaling and reverted the unmarshal changes.